SUILesson1/ContentView.swift

```diff
@@ -7,15 +7,108 @@
 
 import SwiftUI
 
+/// Представление с контентом экрана мини приложения
 struct ContentView: View {
+
+    private enum Constants {
+        static let defaultLabelText = "Текст лейбла для замены"
+        static let showButtonText = "Показать"
+        static let alertExampleText = "Пример алерта"
+        static let sayAText = "Сказать А"
+        static let sayAReplicText = "Аааааааа"
+        static let sayBText = "Сказать Б"
+        static let sayBReplicText = "Б-б-б"
+        static let showActionSheetText = "Показать шторку"
+        static let actionsText = "Действия"
+        static let actionsExemples = "Пример ActionSheet"
+        static let defaultEmptyText = ""
+    }
+
+    // MARK: - private properties
+
+    @State private var isErrorFirst = false
+    @State private var isErrorSecond = false
+    @State private var isPresented = false
```

Comment:
```suggestion
    @State private var isErrorFirstShown = false
    @State private var isErrorSecondShown = false
    @State private var isPresentedShown = false
```
если флаг что-то показывает, то на конце пишется Shown